src/index.js

```diff
@@ -147,6 +147,8 @@ document.addEventListener("DOMContentLoaded", () => {
     const id = Number(task.dataset.id);
     database.toggleDone(id, isDone, () => {
       task.classList.toggle("is-done", isDone);
+      task.classList.toggle("is-primary", !isDone);
+      task.classList.toggle("is-secondary", !isDone);
```

Comment:
There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"? 

Also, there seems to be a little bug here, putting "!isDone" in both toggle functions will remove and add both classes at the same time.